src/containerapp/azext_containerapp/commands.py

```diff
@@ -174,3 +174,6 @@ def load_command_table(self, _):
         g.custom_command('bind', 'bind_hostname', exception_handler=ex_handler_factory())
         g.custom_command('list', 'list_hostname')
         g.custom_command('delete', 'delete_hostname', confirmation=True, exception_handler=ex_handler_factory())
+
+    with self.command_group('containerapp compose', is_preview=True) as g:
```

Comment:
The containerapp extension as a whole is in preview, so I don't think we need this `is_preview=True` here

Reply:
Removed `is_preview=True`